nexus-repository-helm/src/main/java/org/sonatype/repository/helm/internal/content/recipe/HelmContentFacetImpl.java

```diff
@@ -52,9 +53,10 @@
   private final IndexYamlAbsoluteUrlRewriter indexYamlAbsoluteUrlRewriter;
 
   @Inject
-  public HelmContentFacetImpl(@Named(HelmFormat.NAME) final FormatStoreManager formatStoreManager,
-                              final HelmAttributeParser helmAttributeParser,
-                              final IndexYamlAbsoluteUrlRewriter indexYamlAbsoluteUrlRewriter)
+  public HelmContentFacetImpl(
```

Comment:
Minor: redundant refactoring

Reply:
Suggested by IDE. is it ok if we keep so?

Reply:
It's a minor comment so it's up to you. I'd suggest using "VCS only" in the refactoring menu